packages/webcomponents/src/ui-components/form/form-control-radio.tsx

```diff
@@ -29,78 +28,73 @@ export class RadioInput {
   @Prop() name: any;
   @Prop() value: string;
   @Prop() helpText?: string;
-  @Prop() defaultValue?: string;
   @Prop() inputHandler: (name: string, value: string) => void;
   @Prop() disabled: boolean;
   @Prop() errorText: string;
   @Prop() checked: boolean;
 
-  @Watch('defaultValue')
-  handleDefaultValueChange(value: string) {
-    this.updateInput(value);
-  }
-
   @Event() formControlInput: EventEmitter<any>;
   @Event() formControlBlur: EventEmitter<any>;
 
-  componentDidLoad() {
-    this.updateInput(this.value);
-  }
-
   handleFormControlInput = (event: any) => {
     const target = event.target;
     const name = target.getAttribute('name');
-    this.inputHandler(name, target.value);
-    this.formControlInput.emit({ name, value: target.value });
+    this.inputHandler(name, this.value);
+    this.formControlInput.emit({ name, value: this.value });
```

Comment:
This is unrelated to this specific PR - but I wanted to note it because `handleFormControlInput` can probably be shortened to something like the following on each form-control input component

```
handleFormControlInput = (event: any) => {
    const value = event.target.value;

    this.inputHandler(this.name, value);
    this.formControlInput.emit({ name: this.name, value: value });
  }
```

IE we don't need to be getting `name` from `event.target` since we are passing `name` as a prop I believe pretty much every time we are rendering any of these form control components

Reply:
I agree with that!